catalogs/catalog-common/src/main/java/org/apache/gravitino/storage/S3Properties.java

```diff
@@ -31,5 +31,8 @@ public class S3Properties {
   // The region of the S3 service.
   public static final String GRAVITINO_S3_REGION = "s3-region";
 
+  // The S3 credentials provider class name.
+  public static final String GRAVITINO_S3_CREDS_PROVIDER = "s3-creds-provider";
+
   private S3Properties() {}
 }
```

Comment:
Why we don't have OSS related properties defined here in catalog-common?

Reply:
The default value of this key is `EC2ContainerCredentialsProviderWrapper` which is NOT `BasicAWSCredentialsProvider`, however, the corresponding value for `OSS` is the one that uses AS/SK, so we do not need to add this one.


Another problem, we use 

the key `s3-creds-provider` in Gravitino server and use `s3.creds.providers` for GVFS, is that OK?

Reply:
> he key s3-creds-provider in Gravitino server and use s3.creds.providers for GVFS(replace '-' with '.', other keys are similar), is that OK?

I'm OK with this.